ts/Core/Series/SeriesDefaults.ts

```diff
@@ -2548,6 +2548,33 @@ const seriesDefaults: PlotOptionsOf<Series> = {
      * @apioption plotOptions.series.legendSymbol
      */
 
+    /**
+     * Defines the color of the legend symbol for this series.
```

Comment:
I would describe more, e.g. what happens when it is `undefined`. This also does not work in styled mode, maybe the information would be useful.

Reply:
Thanks for the suggestion, applied